src/effects/Reverb.cpp

```diff
@@ -367,38 +439,23 @@ bool EffectReverb::TransferDataToWindow(const EffectSettings &)
    return true;
 }
 
-bool EffectReverb::TransferDataFromWindow(EffectSettings &)
-{
-   auto& rs = mSettings;
-
-   rs.mRoomSize = mRoomSizeS->GetValue();
-   rs.mPreDelay = mPreDelayS->GetValue();
-   rs.mReverberance = mReverberanceS->GetValue();
-   rs.mHfDamping = mHfDampingS->GetValue();
-   rs.mToneLow = mToneLowS->GetValue();
-   rs.mToneHigh = mToneHighS->GetValue();
-   rs.mWetGain = mWetGainS->GetValue();
-   rs.mDryGain = mDryGainS->GetValue();
-   rs.mStereoWidth = mStereoWidthS->GetValue();
-   rs.mWetOnly = mWetOnlyC->GetValue();
-
-   return true;
-}
```

Comment:
I see a deleted function above but I don’t see the ValidateUI that should replace it in the second step.

Reply:
Hmm no, I do not delete TransferDataFromWindow here, I only replace all occurrences of mParams with local reference rs.

Reply:
In step 2, you can see that the contents of this method has moved to ValidateUI

Reply:
I see it now.  I overlooked it.

If you don't copy and paste the function to a different place in the file, it could help make the commit more easily reviewable.